doc/Tex/RAM_SCB_scripts.tex

```diff
@@ -1,73 +1,39 @@
-RAM-SCB comes packaged with many helpful scripts that aid the user.  These 
-scripts are written either in Perl or Python with the standard libraries only.
-This allows them to be run on many systems without needing to install 
-additional software as both languages are ubiquitous in Unix-like 
-environments.  The subdirectory {\tt Scripts} contains several helpful 
-scripts that are unique to RAM-SCB while the subdirectory {\tt Share/Scripts}
-contains scripts designed for SWMF-like applications but often useful for 
-RAM-SCB as well.
+RAM-SCB comes packaged with many helpful scripts that aid the user.  These scripts are written either in Perl or Python with the standard libraries only. This allows them to be run on many systems without needing to install additional software as both languages are ubiquitous in Unix-like environments. The subdirectory {\tt Scripts} contains several helpful scripts that are unique to RAM-SCB while the subdirectory {\tt Share/Scripts} contains scripts designed for SWMF-like applications but often useful for RAM-SCB as well.
 
-All scripts are designed to be called from the command line and have a common
-help feature:
+All scripts are designed to be called from the command line and have a common help feature:
 \begin{verbatim}
 ScriptName -h
 \end{verbatim}
-\noindent will print out the script's help text.  This info is typically 
-more complete and up-to-date than the info listed here.
+\noindent will print out the script's help text.  This info is typically more complete and up-to-date than the info listed here.
 
-While all RAM-SCB scripts are listed here, only a handful of SWMF scripts are
-described.  Be sure to explore the {\tt Share/Scripts} directory before you
-constructing your own solutions to common problems (e.g. endian conversions, 
-PARAM-checking, etc.) 
+While all RAM-SCB scripts are listed here, only a handful of SWMF scripts are described. Be sure to explore the {\tt Share/Scripts} directory before you constructing your own solutions to common problems (e.g. endian conversions, PARAM-checking, etc.) 

```

Comment:
This section also needs to have the `updateRAMindices` script. Not sure we need to document the "rt"-oriented scripts, although we should check whether everything is Python3 compatible.

Reply:
Good point